content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
+defined by conditional rules. *Coupon Programs* are configured in the :menuselection:`Sales` or
+:menuselection:`Website` applications.
+
+.. note::
+   The :menuselection:`eCommerce` module must be installed in order to create coupon codes from the
+   :menuselection:`Website` application.
+
+.. image:: after_sales/after-sales-coupon-setup.png
+   :align: center
+   :alt: View of a coupon program configuration page.
+
+To generate a coupon, open a :menuselection:`Helpdesk` :guilabel:`Ticket` and click on the
+:guilabel:`Coupon` button in the upper left corner. Select an option from the :guilabel:`Coupon
+Program` drop-down menu. Click :guilabel:`Generate`.
+
+.. image:: after_sales/after-sales-generate-coupon.png
+   :align: center
+   :alt: View of a coupon generation window.
+
+The :guilabel:`Coupon Code` can be copied directly from the pop-up window (by clicking the
+:guilabel:`Copy` button), or sent in an email by clicking :guilabel:`Send`.
+
+.. note::
+   All followers of the :guilabel:`Ticket` will be added as recipients to the email. Additional
+   recipients can be added to the email as well.
+
+   .. image:: after_sales/after-sales-coupon-email.png
+      :align: center
+      :alt: View of an email draft window with coupon code.
+
+Once a :guilabel:`Coupon Code` has been generated, a :guilabel:`Coupons` smart button will be added
+to the top of the :menuselection:`Helpdesk` :guilabel:`Ticket`. Click here to view the coupon code,
+expiration date, and additional information.
+
+.. image:: after_sales/after-sales-coupon-smart-button.png
+   :align: center
+   :alt: View of the smart buttons on a ticket focusing on the coupon button.
+
+.. seealso::
+   `After-Sales Features <https://www.odoo.com/slides/slide/after-sales-features-1699>`_
+
+Facilitate a product return with a reverse transfer
+===================================================
+
+Returns are completed through *reverse transfers*, which generate new warehouse operations for the
+returning products. Click the :guilabel:`Return` button at the top left of a :guilabel:`Ticket` to
+open the :guilabel:`Reverse Transfer` pop-up window.
+
+.. image:: after_sales/after-sales-return-button.png
    :align: center
+   :alt: View of a Helpdesk ticket with the return button highlighted.
 
-Generate credit notes from tickets
-----------------------------------
+.. note::
+   The :guilabel:`Return` button will only appear on a :guilabel:`Ticket` if the customer selected
+   has recorded deliveries in the database.
 
-You can use a credit note to refund a customer or adjust the amount due. For that, simply go to
-your ticket page, click on *Refund* and select the corresponding *Invoice*. Clicking on *Reverse*
-generates a credit note, and you can *Post* it while still being in the *Helpdesk* app.
+By default, the quantity will match the validated quantity from the delivery order. Update the
+:guilabel:`Quantity` field if necessary.
```

Comment:
might be better as a **note**

Reply:
I thought the same at first. However, since there is already a note in that section, it seems cluttered. Plus, this section is pretty light on instructions (because there really are only two things to do-change the quantity and click return), so i would rather leave it as a step.